packages/ui/components/FormWrapper/FormWrapper.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import CheckboxGroup from '@refract-ui/hook-fields/CheckboxGroup';
+import FieldErrors from '@refract-ui/hook-fields/FieldErrors';
+import Select from '@refract-ui/hook-fields/Select';
+import TextInput from '@refract-ui/hook-fields/TextInput';
+import s from '@refract-ui/sc';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import type { FormWrapperProps } from '.';
+import FormWrapper from '.';
+
+const meta: Meta<FormWrapperProps> = {
+  title: 'ui/FormWrapper',
+  component: FormWrapper,
+  parameters: {
+    layout: 'centered'
+  },
+  tags: ['autodocs']
+};
+
+export default meta;
+
+type Story = StoryObj<FormWrapperProps>;
+
+function FormFields() {
+  const {
+    formState: { errors }
+  } = useFormContext();
+
+  const selectOptions = [
+    { value: '', label: 'Select option...', disabled: true },
+    { value: '1', label: 'Option 1' },
+    { value: '2', label: 'Option 2' },
+    { value: '3', label: 'Option 3' }
+  ];
+
+  const checkboxOptions = [
+    { value: 'opt1', label: 'Option 1' },
+    { value: 'opt2', label: 'Option 2' },
+    { value: 'opt3', label: 'Option 3' },
+    { value: 'opt4', label: 'Option 4' },
+    { value: 'opt5', label: 'Option 5' }
+  ];
+
+  const Container = s.div`
+  & > div { 
+  margin-bottom: 1rem;
+  }
+  `;
+
+  return (
+    <Container>
+      <Controller
```

Comment:
Rendering the inputs with `Controller` was the only way I could get the refract `hook-fields` to pass data & work with error detection. 
It seems like ideally you wouldn't need to use a `Controller` and could just do:
`<TextInput label="name" id="name" {...register('name', { required: true })}/>`